pwa/src/styles/main.css

```diff
@@ -219,3 +219,24 @@ a:hover {
     border: none !important;
     background: inherit !important;
 }
+
```

Comment:
All the required styling can go to the created `.css` file mentioned earlier (should be located somewhere like `/components/loadingOverlay/loadingOverlay.css`).

To actually use this styling within your new component, you can include it like so (this code goes in `loadingOverlay.tsx`):

`import "./loadingOverlay.css";`

And to scope the `.css` code you can use another prefix (you're using `spinner-overlay` now, which is fine but can be more specific):

```
.loadingOverlay {
  ...
}

.loadingOverlay > .loadingOverlay-content {
  ...
}

.loadingOverlay > ... // always prefix with .loadingOverlay so that you can never style another component by accident
```